Makefile

```diff
@@ -64,8 +64,10 @@ completions:
 	mkdir -p completions/fish/vendor_completions.d
 	register-python-argcomplete --shell fish ramalama > completions/fish/vendor_completions.d/ramalama.fish
 
-	mkdir -p completions/zsh/vendor-completions
-	register-python-argcomplete --shell zsh ramalama > completions/zsh/vendor-completions/_ramalama
+	if ! (register-python-argcomplete --shell zsh ramalama > completions/zsh/vendor-completions/_ramalama); then \
```

Comment:
How about?
```
	tmpfile=$(mktemp)
	if ! (register-python-argcomplete --shell zsh ramalama > $i{tmpfile}); then
		mkdir -p completions/zsh/vendor-completions; \
		mv ${tmpfile} completions/zsh/vendor-completions/_ramalama; \
	else
		rm ${tmpfile}
	fi
```

Reply:
I put in some Makefile equivalent that seems to work